lib/pgfinder/matching.py

```diff
@@ -485,3 +487,63 @@ def add_most_likely_structure(group):
     merged_df = pd.concat([most_likely, unmatched_rows])
 
     return merged_df.reset_index(drop=True)
+
+
+def consolidate_results(
+    df: pd.DataFrame,
+) -> pd.DataFrame:
+    """Add a final table of muropeptide structures and their relative abundances
+
+    Parameters
+    ----------
+    df: pd.DataFrame
+        DataFrame of structures to be processed.
+
+    Returns
+    -------
+    pd.DataFrame
+        The input dataframe with additional columns containing the consolidated results
+    """
+    INTENSITY_COLUMN = "Intensity (consolidated)"
```

Comment:
Not a fan of having values hard-coded, I think its better to set them as arguments with defaults.

Reply:
I've never loved having these as arguments, and I suppose ideally I think they should be global constants (or at least set in a global config some place), since I can't think of any reason to change from these defaults and they add a lot more complexity to the function signatures, but I agree that where things aren't just hard-coded strings, they do seem to be default arguments, so I've made that change to keep things more consistent across the codebase!